include/seqan3/argument_parser/validators.hpp

```diff
@@ -600,10 +600,16 @@ template <typename file_t = void>
 class output_file_validator : public file_validator_base
 {
 public:
+    //!\brief Stores, if it is valid to overwrite the output file.
+    bool allow_overwrite = false; // ToDo: bool oder enum? Für jedes File einzeln entscheiden, also zum Pfad dazugeben? Oder als Vektor angeben?
```

Comment:
`append` cannot be implemented with the argument parser, as the user receives the file name and opens the file themself (with or without append flag). So I suggest to only focus on the binary distinction of the output file "may exist or not". 

**Note**: If we would not handle it at all, the decision (of throwing an error if the file exists) could be still be made by the app developer after obtaining the filename.